best-practices/content/10_IAM.tex

```diff
@@ -1,13 +1,128 @@
 
-\section{IAM Varianten}
+\section{\acrshort{iam} Varianten}
+Eine \acrfull{iam} ist ein zentrales System welches die Entitäten, wie Benutzeraccounts und Computer, erfassen kann. Zudem können Berechtigungen auf Entitäten in einer Domäne verwaltet.\\
 
+%TODO: style correct @sevigrimm should use quote or some fancy style?
+\textbf{Nun kommt die Frage auf: Geht es nicht ohne?}\\
+Es ist grundsätzlich möglich ohne ein IAM System zu arbeiten, wobei der Aufwand für das Management der Userverwaltung auf allen Maschinen schnell den Rahmen des Budgets sprengt.
+Der allgemeine Konsenus ist dass ab 15 aktiven Bentuzern ein IAM verwendet werden soll.\\
+
+Eine IAM Lösung bringt viele Vorteile, gleichzeitig birgt es auch Risiken.\\
+
+\begin{minipage}{\linewidth}
+    \begin{multicols}{2}
+        \begin{table}[H]
+            \begin{center}
+                \textbf{Vorteile:}
+                \begin{itemize}
+                    \item zentrales Accountmanagement
+                    \item zentrales Accessmanagement
+                    \item schnelle Skalierung Server/Client Umgebung
+                    \item single Source of Truth\footnote{\href{https://de.wikipedia.org/wiki/Single\_Point\_of\_Truth}{https://de.wikipedia.org/wiki/Single\_Point\_of\_Truth}}
+                    \item Policysystem zur Konfiguration von Client
+                    \item meist gebündelt mit nützlichen Features\footnote{DHCP, DNS, NTP, CA etc. (Produktabhängig)}
+                    \item Single Sign-On
+                \end{itemize}
+            \end{center}
+            \caption{Vorteile IAM}
+        \end{table}
+        \begin{table}[H]
+            \begin{center}
+                \textbf{Nachteile:}
+                \begin{itemize}
+                    \item hoher initial Aufwand
+                    \item höchste Priorität eines Angreifers
+                    \item anfällig auf Misskonfigurationen
+                    \item abhängigkeit eines stabilen Systems
+                \end{itemize}
+            \end{center}
+            \caption{Nachteile IAM}
+        \end{table}
+    \end{multicols}
+\end{minipage}
+%TODO: @sevigrimm ergänzungen?
+%TODO: Add glossary for DHCP DNS NTP CA
+
+\vspace{1cm}
+Wie bei vielem ist es wichtig das richtige Produkt für seine Umgebung zu finden. Als nächstes werden zwei, im Markt beliebte Produkte vorgestellt.
+
+\subsection{Active Directory}
+Active Directory ist ein Produkt von Microsoft und kann exklusiv auf einem Microsoft Windows Server installiert werden. Active Directory ist einfach zu installieren, wobei die Konfiguration manchmal Herausforderungen birgt.\\
+
+Active Directory ist mit mehr als 80\% Marktanteil der Marktführer. Die Active Directory Funktion wird mit DNS, DHCP und NTP gebündelt. Das Produkt bietet alle grundlegenden Funktionen wie User-, Geräteverwaltung, DHCP, DNS, NTP und viele weitere einfache, bis zu komplexe Funktionen. Vorallem in Windowsnetzwerken ist die Integration sehr gut und intuitiv.\\
```

Comment:
```suggestion
Active Directory ist mit mehr als 80\% Marktanteil der Marktführer. 
Das Produkt bietet alle grundlegenden Funktionen wie User-, Geräte- und Berechtigungsverwaltung.
Ausserdem lässt sich Active Directory integrieren mit anderen Windows Server Funktionen wie DHCP, DNS, NTP und weiteren externen Funktionen von anderen Anbietern. 
Vorallem in Windowsnetzwerken ist die Integration sehr gut und intuitiv.\\
```

Reply:
@sevigrimm should I accept here or should we clarify a little?